src/App.tsx

```diff
@@ -1,12 +1,21 @@
-import Home from 'pages/Home';
 import React from 'react';
 import GlobalStyles from 'styles/GlobalStyles';
+import DataInput from 'components/molecules/DateInput';
 
 function App(): JSX.Element {
   return (
     <>
       <GlobalStyles />
-      <Home />
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          height: ' 100vh',
+        }}
+      >
+        <DataInput label="Label name" isOpen onClick={(date) => console.log(date)} />
```

Comment:
I think we should have props value here, which specify current date in initial state. Maybe this is not needed. To discuss